main.cpp

```diff
@@ -117,6 +120,91 @@ Thing 10) Audio Mixer
     3) monitor input through headphones 
  */
 
+/*
+Thing 5) Equalizer
+5 properties:
+    1) high pass switch 
+    2) low frequency controls
+    3) low-mid frequency controls
+    4) mid frequency controls
+    5) high frequency controls
+3 things it can do:
+    1) lower frequencies 
+    2) boost frequencies
+    3) cut frequencies
+ */
+
+/*
+Thing 6) Compressor
+5 properties:
+    1) compressor switch button
+    2) threshold knob
+    3) gain reduction meter
+    4) atack
+    5) sustain
+3 things it can do:
+    1) add sustain to control dynamics
+    2) make weak signals louder
+    3) compress loud signals
+ */
+
+/*
+Thing 7) Input controls
+5 properties:
+    1) hi-z selector switch
+    2) line selector switch
+    3) 48v selector switch
+    4) high pass filter switch
+    5) gain knob
+3 things it can do:
+    1) select input
+    2) turn on phanto power
+    3) adjust input gain
+ */
+
+/*
+Thing 8) Routing controls
+5 properties:
+    1) fader
+    2) post fader selector switch
+    3) master selector switch
+    4) bus selector switch
+    5) mute
```

Comment:
Fader is an object with sub properties.   you'll want a property, like `gain level` or `send level`
 - [ ] revise